public/index.html

```diff
@@ -0,0 +1,171 @@
+          <p class="feature-list-subheading">Example of Construction</p>
+        </div>
+        <div class="feature-list-card-container container">
+          <div class="feature-list-img-card">
+            <img class="card-img" src="/public/assets/1.jpg" alt="house 1" />
+            <h3 class="card-title">東京都XX様宅</h3>
+            <p class="card-description">ここに説明を入れます。サンプルテキスト。</p>
+            <button class="card-btn btn-1">詳しくみる</button>
+          </div>
+          <div class="feature-list-img-card">
+            <img class="card-img" src="/public/assets/2.jpg" alt="house 1" />
+            <h3 class="card-title">東京都XX様宅</h3>
+            <p class="card-description">ここに説明を入れます。サンプルテキスト。</p>
+            <button class="card-btn btn-2">詳しくみる</button>
+          </div>
+          <div class="feature-list-img-card">
+            <img class="card-img" src="/public/assets/3.jpg" alt="house 1" />
+            <h3 class="card-title">東京都XX様宅</h3>
+            <p class="card-description">
+              ここに説明を入れます。サンプルテキスト。ここに説明を入れます。サンプルテキスト。ここに説明を入れます。サンプルテキスト。
+            </p>
+            <button class="card-btn">詳しくみる</button>
+          </div>
+        </div>
+      </section>
+      <section class="commerce-info container-commerce container section">
+        <div class="whats-new">
+          <h1>新着情報</h1>
+          <hr />
+          <div class="commerce-info-row">
+            <time class="commerce-dates" datetime="2023-11-17">2023/11/17</time>
+            <div class="commerce-tags tag1">お知らせ</div>
+            <p class="commerce-description">
+              ハウスメーカー・工務店向け無料ホームページテンプレート tp_housebuilder1公開。
+            </p>
+          </div>
+          <div class="commerce-info-row">
+            <time class="commerce-dates" datetime="2023-11-17">20XX/00/00</time>
+            <div class="commerce-tags tag2">施工事例</div>
+            <p class="commerce-description">サンプルテキスト。サンプルテキスト。サンプルテキスト。</p>
+          </div>
+          <div class="commerce-info-row">
+            <time class="commerce-dates" datetime="2023-11-17">20XX/00/00</time>
+            <div class="commerce-tags tag3">キャンペーン</div>
+            <p class="commerce-description">サンプルテキスト。サンプルテキスト。サンプルテキスト。</p>
+          </div>
+          <div class="commerce-info-row">
+            <time class="commerce-dates" datetime="2023-11-17">20XX/00/00</time>
+            <div class="commerce-tags tag4">お知らせ</div>
+            <p class="commerce-description">サンプルテキスト。サンプルテキスト。サンプルテキスト。</p>
+          </div>
+          <div class="commerce-info-row">
+            <time class="commerce-dates" datetime="2023-11-17">20XX/00/00</time>
+            <div class="commerce-tags tag5">施工事例</div>
+            <p class="commerce-description">
+              サンプルテキスト。サンプルテキスト。サンプルテキスト。サンプルテキスト。サンプルテキスト。サンプルテキスト。サンプルテキスト。サンプルテキスト。サンプルテキスト。サンプルテキスト。サンプルテキスト。サンプルテキスト。
+            </p>
+          </div>
+        </div>
+        <div class="template-info">
+          <h1>テンプレートのご利用前に必ずお読み下さい</h1>
```

Comment:
[should]
Adding class name would be better

Reply:
@sharray2918 
Thank you for pointing this out, I have switched from targeting the elements
to using class names.